### DISCUSSION TITLE
Add Icelandic to the translation project

### DISCUSSION THREAD

**Discussion by TomicekRosic** [iSoron/uhabits#1782] [Feature Requests] [answered]
I'd like to translate the app to Icelandic, thanks. I've already joined it as strikeCunny2245

On the discussion page I noticed that people have asked for Bengali, Gujarati, Malayalam, and Georgian between 2 years to a month ago without answers, so I figured I'd ask here instead

**Comment by hiqua** [comment, marked as answer]
I have the rights on crowdin so I took the liberty of adding some of these just now. Not sure if something else is needed, please give it a try.

@iSoron FYI.

**Comment by TomicekRosic** [comment]
Thank you!
